operators/pkg/instctrl/const.go

```diff
@@ -29,4 +29,14 @@ const (
 	EvEnvironmentErr = "EnvironmentEnforcementFailed"
 	// EvEnvironmentErrMsg -> the event message corresponding to a failed environment enforcement.
 	EvEnvironmentErrMsg = "Failed to enforce environment %v"
+	// PVC name
+	UserPvcName = "user-pvc"
+	// NFS secret name
+	NFSSecretName = "user-pvc-secret"
+	// NFS Server key in NFS secret
+	NFSSecretServerNameKey = "server-name"
+	// NFS path key in NFS secret
+	NFSSecretPathKey = "path"
+	// The path where to mount the NFS volume
+	NFSMountPath = "/media/MyDrive"
```

Comment:
Constants shall not be duplicated among controllers, otherwise they become useless.

Reply:
Removed unnecessary ones